src/core/evaluator.js

```diff
@@ -3709,6 +3742,12 @@ class PartialEvaluator {
           toUnicode,
         };
         const widths = dict.get("Widths");
+        const standardFontName = getStandardFontName(baseFontName);
+        let file = null;
+        if (standardFontName) {
+          file = await this.fetchStandardFontData(standardFontName);
+          properties.isStandardFont = true;
```

Comment:
If loading of the standard font file failed, and thus `file === null`, it probably doesn't make sense to set the `isStandardFont` property since that affects glyph mapping!?

```suggestion
          properties.isStandardFont = !!file;
```